.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "m2e-core-tests"]
 	path = m2e-core-tests
-	url = https://github.com/tesla/m2e-core-tests.git
-	branch = master
+	url = https://github.com/HannesWell/m2e-core-tests.git
```

Comment:
don't forget to revert this :-)

Reply:
Thanks for the reminder. Not sure that I would not have missed it. :D
Done.